pkgs/by-name/ca/cargo-3ds/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-3ds";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "rust3ds";
+    repo = pname;
```

```suggestion
    repo = "cargo-3ds";
```
Do not superfluously reuse `pname`

This is considered a bad practice, as discussed in https://github.com/NixOS/nixpkgs/issues/277994.

---

Oh, I did not know. Thank you!